src/js/player.js

```diff
@@ -3991,6 +4005,9 @@ class Player extends Component {
      */
     this.trigger('error');
 
+    // notify hooks of the per player error
+    hooks('error').forEach((hookFunction) => hookFunction(this, this.error_));
```

Comment:
we should use the `err` here so that a hook function does not modify player.error_ and cause weird things to happen.

Reply:
`player.error()` returns `player.error_` directly, so, I don't see an issue with returning it here too.